mango/Makefile

```diff
@@ -0,0 +1,27 @@
+# Definitions
+build_output = mango
+build_tool = runtime-container.DONE
+build_number ?= none
+git_commit ?= $(shell git log --pretty=oneline -n 1 -- ../mango ../spark-and-maven | cut -f1 -d " ")
```

Comment:
This gives you the git commit from `../spark-and-maven` instead of `../mango`. 

`git_commit ?= $(shell git log --pretty=oneline -n 1 -- ../mango | cut -f1 -d " ")`